tst/EnergyPlus/unit/SurfaceGeometry.unit.cc

```diff
@@ -5841,6 +5846,77 @@ TEST_F(EnergyPlusFixture, HeatBalanceIntRadExchange_SetupEnclosuresWithAirBounda
     EXPECT_EQ(state->dataHeatBal->space(1).HTSurfaceLast, Zone1Floor);
     EXPECT_EQ(state->dataHeatBal->space(2).HTSurfaceLast, Zone2Floor);
     EXPECT_EQ(state->dataHeatBal->space(3).HTSurfaceLast, Zone3Floor);
+
+    // Check MRT calculations
```

Comment:
Extend existing unit test to check MRT calcs. All other unit test changes are just to keep things working.